typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumFormFORMWarpConfig.ts

```diff
@@ -0,0 +1,39 @@
+import { ethers } from 'ethers';
+
+import { ChainMap, HypTokenRouterConfig, TokenType } from '@hyperlane-xyz/sdk';
+
+import {
+  RouterConfigWithoutOwner,
+  tokens,
+} from '../../../../../src/config/warp.js';
+
+const FormSafes = {
```

Comment:
nit: would probably choose either SNAKE_CASE or camelCase for this, PascalCase feels out of pattern